compiler/rustc_borrowck/src/lib.rs

```diff
@@ -1149,6 +1161,61 @@ impl<'a, 'tcx> MirBorrowckCtxt<'a, '_, 'tcx> {
         error_reported
     }
 
+    /// Through #123739, backward incompatible drops (BIDs) are introduced.
```

Comment:
@dingxiangfei2009 A tiny late nit. I have just learned from this comment what `BID` stand for. Is it a commonly used acronym? I wasn't able to find it using a search engine. Could only find it disentangled here.

(thanks @lqd for the pointing me here)